format/writers.go

```diff
@@ -31,8 +31,9 @@ func NewFileWriter(filePath string) io.Writer {
 	return &fileWriter{filePath}
 }
 
+//nolint:gosec,gomnd // need to test this in a future PR
```

Comment:
can you please create a bug/pbi and then just put AB#12345 to link to it. We try to make sure TODO's have future backlog captured so it doesn't disappear longterm.

Reply:
https://thycotic.visualstudio.com/Delinea.Work/_workitems/edit/561861